internal/context.go

```diff
@@ -0,0 +1,61 @@
+package internal
```

Comment:
I believe the usage of `internal` is not like this but create `internal/xxx` for subpkg